src/lsp_extension.ts

```diff
@@ -0,0 +1,9 @@
+import { Hover, Position, Range, RequestType, TextDocumentIdentifier, WorkDoneProgressParams } from "vscode-languageclient";
```

Comment:
We could name the file somethin like `hover_extension.ts` ?

Reply:
done